lib/paged.rb

```diff
@@ -0,0 +1,22 @@
+require 'ostruct'
+
+module Paged
```

Comment:
There is already `Paginate` interactor and `represent_paginated`. Why the hell there is a _third_ piece of code to do this – without explanation?